content/pages/employment/index.md

```diff
@@ -40,3 +40,43 @@ majorlinks:
 We can support you in all stages of your job search. We’ve teamed up with the Department of Labor to give you career advice, help building your résumé, and access to employers who want to hire Veterans and military spouses.
 
 </div>
+
+<div class="va-introtext">
+
+With VA health care, you’re covered for regular checkups with your primary care provider and appointments with specialists (like cardiologists, gynecologists, and mental health providers). You also gain access to home health and geriatric (elder) care, medical equipment, prosthetics, and prescriptions.
+
+</div>
+
+<div class="va-alert usa-alert usa-alert-warning">
+  <div class="usa-alert-body">
+    <h5 class="va-alert-title">Are you homeless or at risk of becoming homeless?<br><a id="crisis-expander-link">VA may be able to help</a>.
+    </h5>
+    <div id="crisis-expander-content" class="expander-content expander-content-closed">
+      <div class="expander-content-inner">
+      <br>
+        <p>We offer many programs and services—including education and job training—to support Veterans who are homeless or at risk of becoming homeless. We can also connect you with other non-VA resources in your community. Find the support you need by: </p>
+        <ul>
+          <li>Calling the National Call Center for Homeless Veterans at 877-4AID-VET (877-424-3838) for help 24 hours a day, 7 days a week, <b>or</b></li>
+          <li><a href="https://www.veteranscrisisline.net/ChatTermsOfService.aspx?account=Homeless%20Veterans%20Chat">Chatting online with a trained VA staff member</a>, <b>or</b></li>
+          <li>Getting job-placement help from a Community Employment Coordinator (CEC). <a href='http://www.va.gov/homeless/cec-contacts.asp'>Contact a CEC</a></li>
+        </ul>
+        <br>
+        <p><b>If you’re a female Veteran, you can also:</b></p>
+        <ul>
+          <li><a href="/facilities/">Contact your closest VA medical center</a>.<br>
+      	(Ask for the female program manager for help getting housing, health care, and other benefits.)</li>
+      	  <li><a href="http://nchv.org/index.php/help/help/women_veterans/">Find a women’s shelter in your state</a>.</li>
+      	</ul>
+      </div>
+    </div>
+  </div>
+</div>
+
+<script type="text/javascript">
```

Comment:
Is there any particular reason you added the event listener down here instead of inline?

```
<a id="crisis-expander-link" onClick="document.getElementById('crisis-expander-content').classList.toggle('expander-content-closed')">
```

I'm not sure how other modals are handled, but I'd suggest we try to be consistent between inlining the function, a `<script>` tag in the body, and offloading it into a common JS snippet.

Nothing particularly wrong with this, just my ignorance of how it's handled elsewhere.

Reply:
Inline is literally how it's handled everywhere, so we're definitely being consistent, but I can foresee this becoming an issue in the future. It's on my list of things that will need to be standardized!